docs/_theme/section.hbs

```diff
@@ -142,4 +142,11 @@
     <div class='pad1y quiet space-top2 prose-big'>Events</div>
     <%= renderSectionList({ members: section.members.events, renderSection: renderSection, noun: 'Event' }) %>
   <% } %>
+
+  <% if (section.sees) { %>
+    <div class='pad1y quiet space-top2 prose-big'>See</div>
```

Comment:
I think we can come up with a more informative title here, instead of "See". Maybe something along the lines of "As seen in ..." or "Examples that use this method"